From fc146e12898d893612b044cd10f9ef70869a2555 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Tue, 4 Apr 2017 11:16:08 +0100
Subject: [PATCH 1/2] Disable use of unsupported plugins which use geoclue

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
---
 configure.ac        | 8 --------
 plugins/Makefile.am | 2 --
 2 files changed, 10 deletions(-)

diff --git a/configure.ac b/configure.ac
index 7688d87..d257283 100644
--- a/configure.ac
+++ b/configure.ac
@@ -162,17 +162,9 @@ PKG_CHECK_MODULES(COLOR,
 		  colord >= 1.0.2
 		  gnome-desktop-3.0 >= $GNOME_DESKTOP_REQUIRED_VERSION
 		  libcanberra-gtk3
-		  libgeoclue-2.0 >= $GEOCLUE_REQUIRED_VERSION
 		  lcms2 >= $LCMS_REQUIRED_VERSION
 		  libnotify)
 
-PKG_CHECK_MODULES(DATETIME,
-		  libnotify >= $LIBNOTIFY_REQUIRED_VERSION
-		  libgeoclue-2.0 >= $GEOCLUE_REQUIRED_VERSION
-		  geocode-glib-1.0 >= $GEOCODE_GLIB_REQUIRED_VERSION
-		  gweather-3.0 >= $LIBGWEATHER_REQUIRED_VERSION
-		  polkit-gobject-1 >= $POLKIT_REQUIRED_VERSION)
-
 PKG_CHECK_MODULES(DUMMY,
 		  gio-2.0
 		  glib-2.0)
diff --git a/plugins/Makefile.am b/plugins/Makefile.am
index 9324b40..d39ec2e 100644
--- a/plugins/Makefile.am
+++ b/plugins/Makefile.am
@@ -4,8 +4,6 @@ enabled_plugins =	\
 	a11y-keyboard	\
 	a11y-settings	\
 	clipboard	\
-	color		\
-	datetime	\
 	dummy		\
 	power		\
 	housekeeping    \
-- 
2.12.2

