--- gnome-settings-daemon-3.24.0/plugins/common/gsd-device-mapper.c~	2017-03-15 12:07:28.000000000 +0000
+++ gnome-settings-daemon-3.24.0/plugins/common/gsd-device-mapper.c	2017-04-10 00:25:22.671515788 +0000
@@ -975,6 +975,7 @@
 #endif
 }
 
+G_GNUC_INTERNAL
 GsdDeviceMapper *
 gsd_device_mapper_get (void)
 {
@@ -995,6 +996,7 @@
 	return mapper;
 }
 
+G_GNUC_INTERNAL
 void
 gsd_device_mapper_add_input (GsdDeviceMapper *mapper,
 			     GsdDevice	     *device)
@@ -1011,6 +1013,7 @@
 	g_hash_table_insert (mapper->input_devices, device, info);
 }
 
+G_GNUC_INTERNAL
 void
 gsd_device_mapper_remove_input (GsdDeviceMapper *mapper,
 				GsdDevice	*device)
